data/categories.txt

```diff
@@ -5711,7 +5711,7 @@ fr:3Boîte aux lettres|courrier
 de:3Briefkasten|3Postfach|post
 hu:3Postaláda|Levelesláda
 id:Kotak pos
-it:3Casella
+it:3Buca delle lettere|3Buche delle lettere|5Casella della posta|5Casella postale|4Cassetta della posta|4Cassetta postale|4Cassette delle lettere
```

Comment:
You don't need the same 3, 4, or 5 number for words in the same line that have the same prefix. Otherwise users will see more than one suggestion for the same category (e.g. after typing "cass" users will see 3 suggestions at the same time: "Cassetta della posta", "Cassetta postale" and "Cassette delle lettere"

Reply:
Thanks for the review! It should be OK now